doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+.. _rt_industry_setup:
+
+How to launch RTVM with ACRN Industry scenario
+##############################################
+
+Verified version
+****************
+
+- Clear Linux version: **30690**
+- ACRN-hypervisor tag: **acrn-2019w31.5-140000p**
+- ACRN-Kernel(Service VM kernel): **acrn-2019w31.5-140000p**
+- ACRN-Kernel(Preempt-RT kernel): **acrn-2019w31.5-143000p**
+
+Prerequisites
+*************
+Below example is based on Intel Kaby Lake NUC platform with two hard disks,
+SATA disk is for Clear Linux based Service VM and the NVMe disk is for RTVM.
+
+- A Clear Linux build machine (CL ver: 29750) to setup build environment.
+- Intel Kaby Lake (aka KBL) NUC platform with two hard disks inside.
+  (refer to :ref:`the tables <hardware_setup>` for detailed information)
+- Installed Clear Linux OS onto both hard disks on KBL NUC.
+
+.. note::
+      Follow `the installation guide
```

Comment:
@dbkinder , is the alignment acceptable here?

Reply:
Fixed in new release :smile: